include/libopencm3/stm32/g4/crc.h

```diff
@@ -0,0 +1,37 @@
+/** @defgroup crc_defines CRC Defines
+ *
+ * @brief <b>Defined Constants and Types for the STM32G4xx CRC Generator </b>
+ *
+ * @ingroup STM32G4xx_defines
+ *
+ * @version 1.0.0
+ *
+ * @date 18 August 2012
+ *
+ * LGPL License Terms @ref lgpl_license
+ */
+/*
+ * This file is part of the libopencm3 project.
+ *
+ * Copyright (C) 2010 Thomas Otto <tommi@viadmin.org>
```

Comment:
this is incorrect.  IMO, this level of header probably isn't even copyrightable, period, but at the very least, it's not 2010 by thomas otto.

Reply:
I've removed dates and copyright line from all headers I added.